### PR TITLE
Release 4.1.0

### PR DESCRIPTION
**Pull request by cverna**
I did as much cleanup as I could understand :) in the setup.py and spec files.

Signed-off-by: Clement Verna <cverna@tutanota.com>

### CODE REVIEW

File: fedora-packages.spec [hunk 1]
```diff
@@ -1,18 +1,15 @@
-%{!?python_sitelib: %define python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")}
-%{!?pyver: %define pyver %(%{__python} -c "import sys ; print sys.version[:3]")}
-
 %define oldname fedoracommunity
```
Comment (pypingou) [inline fedora-packages.spec]: s/%define/%global

File: fedora-packages.spec [hunk 2]
```diff
@@ -53,72 +50,45 @@ Requires: xapian-bindings-python
 Requires: python-dogpile-core > 0.4.0
 Requires: python-dogpile-cache > 0.4.1
 Requires: python-memcached
-Requires: packagedb-cli
+# Requires: packagedb-cli
```
Comment (pypingou) [inline fedora-packages.spec]: Just drop it :)

File: fedora-packages.spec [hunk 3]
```diff
@@ -53,72 +50,45 @@ Requires: xapian-bindings-python
...
-cp fedoracommunity/widgets/static/javascript/ui/moksha.ui.{grid,popup}.js %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.grid/static/javascript/ui
-%{__mkdir_p} %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.package.updates/static/javascript/
-cp fedoracommunity/widgets/package/static/javascript/bodhi.js %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.package.updates/static/javascript/
-%{__mkdir_p} %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.expander/static/javascript/
-cp fedoracommunity/widgets/static/javascript/jquery.expander.js %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.expander/static/javascript/
-%{__mkdir_p} %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.tree/static/javascript/
-cp fedoracommunity/widgets/static/javascript/jquery.jstree.js %{buildroot}%{_datadir}/%{oldname}/public/toscawidgets/resources/fedoracommunity.widgets.tree/static/javascript/
```
Comment (pypingou) [inline fedora-packages.spec]: All the JS is removed?

**Comment by pypingou** [comment]
Few comments but nothing blocking (maybe just the %define vs %global but it should work regardless)

**Comment by pypingou** [comment]
ie 👍 forme

File: fedora-packages.spec (hunk 3 above)
Comment (cverna) [reply fedora-packages.spec]: The JS is provided by toscawidget (independent rpms). I am not quite why it was packaged in this rpm in the first place. But the app is using the js coming from toscawidget rpms so we can remove these safely.

**Comment by cverna** [comment]
Thanks updated

**Pull request merged**